### PR TITLE
Fixed main function signature in code examples for assignment 1

### PR DESCRIPTION
**Pull request by ChristianMoesl**
@cponcelets I noticed a little bug in the code examples, used in today's homework. It gives a warning during compilation with selfie. So basically it's not a big deal, but just to let you know.

### CODE REVIEW

**Comment by cponcelets** [comment]
I didn't noticed, thank you.

File: manuscript/code/hello-world-minified.c [hunk 1]
```diff
@@ -1 +1 @@
-uint64_t*foo;uint64_t*main(){foo="Hello World!    ";while(*foo!=0){write(1,foo,8);foo=foo+1;}}
\ No newline at end of file
+uint64_t*foo;uint64_t main(){foo="Hello World!    ";while(*foo!=0){write(1,foo,8);foo=foo+1;}}
```
Comment (ckirsch) [inline manuscript/code/hello-world-minified.c]: @ChristianMoesl The intention here is to have zero whitespace...even if there is a compiler warning...

Comment (ChristianMoesl) [reply manuscript/code/hello-world-minified.c]: @ckirsch I see. But why do we have to have zero whitespaces? For Teaching? Isn't this a good example for the limitations of minification?

Comment (ckirsch) [reply manuscript/code/hello-world-minified.c]: @ChristianMoesl To demonstrate that whitespace in C programs has no effect on semantics.

Comment (ChristianMoesl) [reply manuscript/code/hello-world-minified.c]: @ckirsch Ok i see. I'm closing this.

**Pull request closed without merging**